### PR TITLE
Fix #29: Inner class permalinks don't work

### LINKED ISSUE #29: Inner class permalinks don't work
e.g. in the test index, go to http://localhost:3000/project-alpha/alpha.core/Helper and then click on `Utils` in the nav bar, it will take you to http://localhost:3000/project-alpha/alpha.core/Helper.Utils which doesn't work

### PR DESCRIPTION
**Pull request by abesto**
In fact, it wasn't just *perma*links. Resolution for inner classes had some incorrect assumptions.

image

Closes #29

### CODE REVIEW

File: www/components/SPA/SymbolPage.tsx [hunk 1]
```diff
@@ -11,10 +11,28 @@ import ClassContents from "@/components/Sidebar/ClassContents";
...
+    );
+  }
+  return cls;
+}
+
 export default function ModulePage() {
   const { pkg: pkgName, mod: modName, sym: symName } = useParams();
+  if (symName === undefined) {
+    return <div>`Missing "sym" parameter, how did you even get here?`</div>;
+  }
```
Comment (zsol) [inline www/components/SPA/SymbolPage.tsx:35]: <3

**Review by zsol — APPROVED** [review]

File: www/components/SPA/SymbolPage.tsx [hunk 2]
```diff
@@ -11,10 +11,28 @@ import ClassContents from "@/components/Sidebar/ClassContents";
 import ModuleContents from "@/components/Sidebar/ModuleContents";
 import ModuleList from "@/components/Sidebar/ModuleList";
 
+import * as docs from "@/lib/docs";
 import { withoutPrefix } from "@/lib/url";
 
+function resolveClass(mod: docs.Module, name: string): docs.Class | undefined {
+  const parts = withoutPrefix(mod.name, name).split(".");
+  let cls = mod.classes.find(
+    (c) => withoutPrefix(mod.name, c.name) === parts.shift()
```
Comment (zsol) [inline www/components/SPA/SymbolPage.tsx:20]: I come from the future, this code doesn't work. :)

Comment (zsol) [reply www/components/SPA/SymbolPage.tsx:20]: https://github.com/zsol/py.wtf/commit/51aa1ff305a0230461419a705168d95721bda358

Comment (abesto) [reply www/components/SPA/SymbolPage.tsx:20]: Ooooooh. Dammit, and I even had tests! And apparently even those were only for the first class. Dammit.

**Pull request merged**